activerecord/lib/active_record/railtie.rb

```diff
@@ -356,20 +356,21 @@ class Railtie < Rails::Railtie # :nodoc:
     end
 
     initializer "active_record_encryption.configuration" do |app|
-      ActiveSupport.on_load(:active_record) do
-        ActiveRecord::Encryption.configure \
+      ActiveSupport.on_load(:active_record_encryption) do
+        configure(
           primary_key: app.credentials.dig(:active_record_encryption, :primary_key),
           deterministic_key: app.credentials.dig(:active_record_encryption, :deterministic_key),
           key_derivation_salt: app.credentials.dig(:active_record_encryption, :key_derivation_salt),
           **app.config.active_record.encryption
+        )
 
-        auto_filtered_parameters = ActiveRecord::Encryption::AutoFilteredParameters.new(app)
-        auto_filtered_parameters.enable if ActiveRecord::Encryption.config.add_to_filter_parameters
+        auto_filtered_parameters = self::AutoFilteredParameters.new(app)
```

Comment:
Let's keep the full namespace. It is clear

Reply:
I reverted some changes to keep using the full namespace everywhere it was used before.